### DISCUSSION TITLE
Blog Module

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#3521] [General]
I added some improvements to the Blog module today:

https://github.com/oqtane/oqtane.blogs

- upgraded to .NET 8 / Oqtane 5.0
- added Published field to indicate if blog has been published (unpublished blogs are only visible to editors)
- added ability for users to subscribe and be notified when new blogs are published
- notifications are sent by email
- included ability for users to unsubscribe
- added RSS syndication feed
- fixed redirect issue after saving new blog
- made Summary field a textarea element
- fixed form layout issues
- utilized EditMode to manage viewing/editing experience
- improved display of search options

it is running in production here: https://www.oqtane.org/blog

**Comment by sbwalker** [comment]
Version 5.0.0 now released to the Marketplace:

https://www.oqtane.net/

**Reply by leigh-pointer** [reply]
@sbwalker RSS button giving 404 response. on https://www.oqtane.org/blog

**Reply by sbwalker** [reply]
@leigh-pointer I just tried it on Firefox and Edge and it works fine for me. It loads this Url:

https://www.oqtane.org/api/blog/rss/172

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/84a6b6e2-ac50-4261-a2b3-69e259a4e232)

**Reply by leigh-pointer** [reply]
@sbwalker I get the URL https://www.oqtane.org/_rss 
I am not logged in!

**Reply by leigh-pointer** [reply]
@sbwalker using Edge Chrome FireFox Brave errors with 404

**Reply by sbwalker** [reply]
I am not logged in either. Can you please make sure you have the latest version of the page content (maybe you have a cached version or something?)

**Reply by sbwalker** [reply]
@leigh-pointer found the problem - the issue only happens in the blog detail view - not the blog summary view

**Reply by sbwalker** [reply]
@leigh-pointer should be fixed now - let me know

**Reply by leigh-pointer** [reply]
@sbwalker yes, it is all looking good now 👍

**Reply by sbwalker** [reply]
Thank you for letting me know about the issue... I will need to release a 5.0.1 Nuget package

**Reply by sbwalker** [reply]
5.0.1 now available
